sentry-kotlin-multiplatform/src/appleTest/kotlin/io/sentry/kotlin/multiplatform/ApplyKmpEventTest.kt

```diff
@@ -0,0 +1,28 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.kotlin.multiplatform.extensions.applyKmpEvent
+import kotlin.test.Test
+import kotlin.test.assertFalse
+
+class ApplyKmpEventTest {
+    @Test
+    fun `native release is not set if kmp release has same value`() {
+        val nativeEvent = FakeSentryEvent()
+        val kmpEvent = SentryEvent()
+
+        nativeEvent.applyKmpEvent(kmpEvent)
+
+        assertFalse(nativeEvent.wasReleaseSet)
+    }
+}
+
+// init with a SentryLevel as a workaround for:
+// Unable to call non-designated initializer as super constructor
+private class FakeSentryEvent : CocoaSentryEvent(Internal.Sentry.kSentryLevelFatal) {
```

Comment:
can only test cocoa like this since the `SentryEvent` in java is final and we don't use a mocking framework yet